pkg/model/encoding.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+import (
+	"github.com/grafana/tempo/pkg/tempopb"
+
+	"github.com/gogo/protobuf/proto"
+)
+
+// CurrentEncoding is a string representing the encoding that all new blocks should be created with
+const CurrentEncoding = ""
```

Comment:
Shouldn't this encoding be set via config? How is the encoding management expected to be done?

Reply:
I don't think we'll ever set this via config. Like the block encodings we will always just use the latest encoding available. i.e. `CurrentEncoding`.

Reply:
is not something people should care to configure right?  sounds good 👍